tests/approvals/out/unoptimized/ops/ops.awst

```diff
@@ -24,8 +24,7 @@ contract MyContract
   
   test2(size: uint64): void
   {
-    a: bytes = bzero(size)
-    assert(len(a) == size)
+    assert(len(bzero(size)) == size)
```

Comment:
This seems a bit unusual, that the assignment to `a` has been removed entirely before any optimisations

Reply:
https://github.com/algorandfoundation/puya-ts/pull/274/commits/69817e64a301d53bda2328581268047f8d39ef59 restores AWST output to be the same as before